src/main/webapp/js/instructorFeedbackEdit/visibilityOptions.js

```diff
@@ -115,10 +115,16 @@ function showVisibilityCheckboxesIfCustomOptionSelected($containingForm) {
 }
 
 var checkCheckboxCallback = function(index, checkbox) {
+    if (checkbox.checked === false) {
```

Comment:
Nothing much, but is it necessary to do strict equality here? (i.e. it may be simplified to `!checkbox.checked`)
Similarly for the `=== true` case.